src/libmongoc/tests/bsonutil/bson-parser.h

```diff
@@ -36,15 +38,13 @@
  * bson_parser_utf8 (parser, "name", &person.name);
  * bson_parser_bool_optional (parser, "hasKids", &person.has_kids);
  * bson_parser_array (parser, "jobs", &person.jobs);
- * bson_parser_doc_alternate (parser, "jobs", &person.jobs);
```

Comment:
In my full implementation of the unified test format, I only used one such `alternate` function. It added complexity to the parsing, so I chose to handle that one case specially.